packages/@aws-cdk/tmp-toolkit-helpers/src/api/io/private/messages.ts

```diff
@@ -81,6 +81,30 @@ export const IO = {
   }),
 
   // 4: Diff (4xxx)
+  CDK_TOOLKIT_I4200: make.debug({
```

Comment:
For all of these: they should have data if useful, or not get a code

Reply:
1) i'm not sure if i should return the formatted diff (string) or the template diff. 
2) why? i see lots of other instances of coded messages that do not have any data attached.